demos/src/demo.scss

```diff
@@ -2,21 +2,112 @@ $o-typography-is-silent: false;
 
 @import '../../main';
 
-.demo-row {
-	clear: both;
+html {
+	@include oColorsFor(page);
+	overflow-y: scroll;
+
+	// type defaults
+	font-family: $o-typography-sans;
+	-ms-text-size-adjust: 100%;
+	-webkit-text-size-adjust: 100%;
+	text-rendering: optimizeSpeed;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+}
+
+.demo {
+	// prevent images and media from expanding its container
+	img,
+	svg,
+	video,
+	object,
+	iframe,
+	figure,
+	figure img,
+	figure svg {
```

Comment:
are you using all these elements in your demo? If not, you don't need all those selectors